### ISSUE TITLE
iOS 6.0: Keyboard navigation using arrow-keys fails in portrait orientation

### ISSUE THREAD

**Issue by kokhoon** [Ranchero-Software/NetNewsWire#3138]
Currently, keyboard navigation from Article to Feeds (with left-arrow), and from Feeds back to Article (with right-arrow), works well on the iPad when it is in landscape orientation (all 3 panels are visible) 

Once the iPad is rotated left or right, keyboard navigation (with left-arrow) from Article will stop at the Timeline panel, it is not possible to get back to Feeds.

With my iPhone in landscape orientation, keyboard navigation (with left-arrow) from articles will stop at the Timeline panel, just like iPad in portrait mode (only Timeline and Article panels are visible).

Keyboard navigation (with left-arrow) does nothing when my iPhone is in portrait orientation

**Comment by brentsimmons** [comment]
Fixed in 7.1.3.
